docs/conf.py

```diff
@@ -30,6 +30,7 @@
 
 # Be strict about any broken references
 nitpicky = True
+nitpick_ignore = []
```

Comment:
Very nice. Thanks for doing it this way. I'll add this line to the skeleton as well for simpler diffs.

Reply:
If you add it to skeleton, you might want to write in a way where `var-annotated` won't trigger when it isn't being re-assigned and the type cannot be inferred.
```py
from __future__ import annotations

...

nitpick_ignore: list[str] = []
```
or 

```
# mypy: disable-error-code="var-annotated"

...

...

nitpick_ignore = []
```
(or disable `var-annotated` in mypy.ini for `docs/conf.py`)

Reply:
Oh! Nice recommendation. Good thing our instincts align. I added jaraco/skeleton@0c326f3 before I saw this. I hope I got the type right. I'm testing it now.